### ISSUE TITLE
[Feature Request] Show cover/album art when playing music

### ISSUE THREAD

**Issue by mmattel** [owncloud/ios#935]
IOS 10.3.3 Client 3.6.2

When download a music track and play it, you get the quicktime symbol as cover/album art.
When you lock the screen during playing, IOS will show you the cover - if embedded in the file.

**Feature Request**: let the client also show the cover/album art when embedded in the file.

**Comment by nasli** [comment]
Would be cool indeed, thanks for your suggestion @mmattel 
This is not in our roadmap for now but contributions are welcome!

**Comment by andrevz** [comment]
Hello @nasli I just created a PR [https://github.com/owncloud/ios/pull/1105](url)  for this feature, I am new to iOS development so any comment or suggest is welcome, please find attached some screen shots of the feature 
![Screen Shot 2019-04-12 at 11 39 41 PM](https://user-images.githubusercontent.com/16736475/56074306-c5108480-5d7d-11e9-9d95-5b8894fffee1.png)
![Screen Shot 2019-04-12 at 11 43 32 PM](https://user-images.githubusercontent.com/16736475/56074308-c9d53880-5d7d-11e9-9f75-00ae0c31d0f1.png)

**Comment by nasli** [comment]
Thank you for your contribution @andrevz, I'm sure a lot of people will love it !!

And congratulations on your journey as iOS dev! 🎉🎉

Glad that you are contributing to this project and for new features at this time ownCloud is focusing the effort on the newest and most modern application you can find [here](https://github.com/owncloud/ios-app). 

I recommend you to keep an eye on it and hope ownCloud can count on you there too.

Check if this feature is not already available in the new app, @michaelstingl or @jesmrec can help you with that.

**Comment by mmattel** [comment]
If this function is in the new app already, pls link.
For the time up to the new app gets released, this would be great to get it in (properly tested of course :smile:)

**Comment by hosy** [comment]
@mmattel this is currently not implemented in the new app. Are you interested to implement this feature? Thank you very much for contributing! The new app is written in Swift.

**Comment by mmattel** [comment]
@hosy thanks for the idea, but I am not able to do this due to completely lack of Swift.
But @andrevz might be able to do so as she has already written that code.

**Comment by andrevz** [comment]
@hosy @mmattel yep I am interested; I am going to try to add the same feature for the new app

**Comment by jesmrec** [comment]
you are very welcome @andrevz. Do not hesitate in asking everything you need.

**Comment by andrevz** [comment]
@jesmrec sure, I was reviewing the new architecture, and have some doubts about where the ui image view should be generated Client or SDK? and How the player is generated? (I lost the track at `DisplayViewController` thank you

**Comment by jesmrec** [comment]
I guess the UI object and player are generated in the client. but @mneuwert, @felix-schwarz and @hosy will answer more accurately. You can open an issue in the `ios-app` repository in which the discussion can go on.
